packages/server-wallet/src/objectives/objective-manager.ts

```diff
@@ -1,21 +1,33 @@
 import {Logger} from 'pino';
 import {BN} from '@statechannels/wallet-core';
+import {Transaction} from 'objection';
 
 import {Bytes32} from '../type-aliases';
-import * as OpenChannelProtocol from '../protocols/open-channel';
-import * as CloseChannelProtocol from '../protocols/close-channel';
+import {
+  getOpenChannelProtocolState,
+  ProtocolState as OpenChannelProtocolState,
+  protocol as openChannelProtocol,
+} from '../protocols/open-channel';
+import {
+  getCloseChannelProtocolState,
+  ProtocolState as CloseChannelProtocolState,
+  protocol as closeChannelProtocol,
+} from '../protocols/close-channel';
 import * as ChannelState from '../protocols/state';
 import {Store} from '../wallet/store';
 import {LedgerRequest} from '../models/ledger-request';
 import {ChainServiceInterface} from '../chain-service';
-import {ProtocolAction} from '../protocols/actions';
+import {FundChannel, SignState, Withdraw} from '../protocols/actions';
 import {recordFunctionMetrics} from '../metrics';
 import {WalletResponse} from '../wallet/response-builder';
 import {Channel} from '../models/channel';
+import {DBObjective} from '../models/objective';
 
 import {ObjectiveManagerParams} from './types';
 import {CloseChannelObjective} from './close-channel';
 
+type SupportedProtocolState = OpenChannelProtocolState | CloseChannelProtocolState;
```

Comment:
Is there a way to avoid the naming collision of 'supported state'? Maybe just use `ProtocolState` as the union type , and put a comment that these are the only ones that are implemented so far.

Reply:
I'm intending to remove this type completely in future refactors.